pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -130,6 +130,9 @@ const (
 	// DeploymentNameVPNSeedServer is a constant for the name of a Kubernetes deployment object that contains
 	// the vpn-seed-server pod.
 	DeploymentNameVPNSeedServer = "vpn-seed-server"
+	// StatefulSetNameVPNSeedServer is a constant for the name of a Kubernetes statefulset object that contains
+	// the vpn-seed-server pods.
+	StatefulSetNameVPNSeedServer = "vpn-seed-server"
```

Comment:
Could you please refer to `DeploymentNameVPNSeedServer` instead of duplicating the constant value? The code already references `DeploymentNameVPNSeedServer` for both `Deployment` and `StatefulSet` (see https://github.com/gardener/gardener/blob/f6efd00a9e6c80a581be33f7269eae316dbef7d9/pkg/component/networking/vpn/seedserver/seedserver.go#L928-L932). Adding another constant is a good idea, but it should take the value from the other constant.

```suggestion
	StatefulSetNameVPNSeedServer = DeploymentNameVPNSeedServer
```